### PR TITLE
Adds a memorial object

### PR DESCRIPTION
**Pull request by Lamrr**
## About the PR 
Adds a generic memorial object, a useful piece of decoration for mappers to spruce up their garden, morgue or engineering department. Name (but not description, see #[2531](https://github.com/space-wizards/RobustToolbox/issues/2531)) can be changed through view variables to add more flavour if wanted, as seen in the screenshot.

Sprite taken from tgmc, and is off-set from the tile a bit which means it can be placed right against walls.

**Screenshots**

Screenshot from 2022-02-11 13-39-46

**Changelog**

:cl:
- add: Commemorate your favourite event with the new memorial object.

### CODE REVIEW

File: Resources/Textures/Structures/Furniture/memorial.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "version":1,
+    "size":{
+        "x":32,
+        "y":64
+    },
+    "license":"CC-BY-NC 3.0",
```
Comment (metalgearsloth) [inline Resources/Textures/Structures/Furniture/memorial.rsi/meta.json:7]:
```suggestion
    "license":"CC-BY-NC-3.0",
```

**Comment by ike709** [comment]
@Lamrr fixed in https://github.com/space-wizards/space-station-14/pull/6648

**Comment by Lamrr** [comment]
> @Lamrr fixed in #6648

Neat, not sure how / if I need to, run another check.

**Comment by ike709** [comment]
Maintainer can re-run checks.

**Comment by metalgearsloth** [comment]
If you merge in master it will re-run + pick up the new licence.

**Pull request merged**